crates/astria-cli/src/sequencer/account.rs

```diff
@@ -48,22 +91,33 @@ struct Create {
 
 impl Create {
     fn run(self) -> eyre::Result<()> {
-        let signing_key = SigningKey::new(OsRng);
+        let bip39_mnemonic =
+            bip39::Mnemonic::new(bip39::MnemonicType::Words12, bip39::Language::English);
+
+        let seed = bip39::Seed::new(&bip39_mnemonic, "");
+        let seed_bytes: [u8; 32] = seed.as_bytes()[0..32]
+            .try_into()
+            .wrap_err("failed to convert seed to 32 bytes")?;
+
+        let signing_key = SigningKey::from(seed_bytes);
+
         let pretty_signing_key = hex::encode(signing_key.as_bytes());
         let pretty_verifying_key = hex::encode(signing_key.verification_key().as_bytes());
 
         let pretty_address: Address = Address::builder()
             .array(signing_key.address_bytes())
             .prefix(&self.prefix)
-            .try_build()?;
+            .try_build()
+            .wrap_err("failed constructing a valid from address from the provided prefix")?;
 
-        println!("Create Sequencer Account");
-        println!();
         // TODO: don't print private keys to CLI, prefer writing to file:
         // https://github.com/astriaorg/astria/issues/594
-        println!("Private Key: {pretty_signing_key}");
-        println!("Public Key:  {pretty_verifying_key}");
-        println!("Address:     {pretty_address}");
+        println!(
```

Comment:
IMO don't indent with spaces here. Just leave it with a single space separator. We need to find a more "sustainable" way to arrange the columns of the output (even beyond tabs, for example).